File: app/src/main/java/com/doubean/ford/data/vo/SizedPhoto.java
package com.doubean.ford.data.vo;

import com.google.gson.annotations.SerializedName;

public class SizedPhoto {
    public String description;
    public String id;
    public SizedImage image;
    public boolean origin;
    @SerializedName("tag_name")
    public String tag;

}

File: app/src/main/java/com/doubean/ford/data/vo/SizedPhoto.kt
package com.doubean.ford.data.vo

import com.google.gson.annotations.SerializedName

class SizedPhoto (
    val id: String,
    val description: String? = null,
    val image: SizedImage? = null,
    val origin: Boolean = false,
    @SerializedName("tag_name")
    val tag: String? = null
)